tests/qchem/openfermion_pyscf_tests/test_dipole_of.py

```diff
@@ -17,6 +17,7 @@
 # pylint: disable=too-many-arguments
 import numpy as np
 import pytest
+from conftest import xfail_on_numpy2  # pylint: disable=no-name-in-module
```

Comment:
Do we need to import with this? I assumed the conftest definitions are implicitly available -- or is that only for fixtures?

Reply:
`xfail_on_numpy2` has been defined as a decorator rather than fixture in `conftest.py` so it still needs to be imported